src/common/System.cpp

```diff
@@ -39,10 +39,8 @@ SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
 #include <sys/random.h>
 #endif
 #ifdef __native_client__
-#if !defined(__saigo__)
-#include <nacl/nacl_exception.h>
-#include <nacl/nacl_minidump.h>
-#endif
+#include <nacl/native_client/src/include/nacl/nacl_exception.h>
```

Comment:
`nacl_exception.h` can be removed. We don't call any of those functions directly. The `nacl_exception` library, however, is needed as a dependency of `minidump_generator`.

Reply:
OK, done.